plugins/services/src/js/components/modals/GroupContainsFrameworksModal.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Modal } from "reactjs-components";
+import ModalHeading from "#SRC/js/components/modals/ModalHeading";
+
+const GroupContainsFrameworksModal = props => {
```

Comment:
Please see other comment about renaming this component.

Reply:
Changed the name